common/types/PerfomanceComment.js

```diff
@@ -5,9 +5,10 @@ let Chance = require("chance")
 
 let PerfomanceComment = T.struct({
   id: Id,
-  perfimanceId: Id,
-  formUserId: Id,
-  toUserId: Id,
+  perfomanceId: Id,
+  fromUserId: Id, // кто написал комментарий
```

Comment:
So it was a typo: `form` -> `from`. I was confused by it.
Russian vs Eng mix again.
Let's communicate here in English!
Agree?